src/components/Create/Create.jsx

```diff
@@ -261,8 +278,8 @@ function Create(props) {
                     </SingleSelect>
                 </Field>
 
-                <Field label="Name the catchment areas" required validationText={nameText} warning>
-                    <Input onChange={handleNameChange}/>
+                <Field label="Name the catchment areas" required validationText={nameText} helpText={`${characterCount}/${maxCharacters} ${i18n.t("characters")}`} warning>
+                    <Input onChange={handleNameChange} value={formInputs.name}/>
                 </Field>
                 <Field label="Select the facility level" required validationText={levelText} error>
```

Comment:
At the moment it's not possible to de-select a facility once one has been selected (though groups can be cleared after they're chosen) -- this might be in a separate issue, but I think it'd be good to be able to de-select a facility.